openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -230,6 +252,86 @@ def handle(self, *args, **options):  # pylint: disable=unused-argument
             except Exception:  # pylint: disable=broad-except
                 log.exception(
                     "Error trying to dump course %s to neo4j, rolling back",
-                    six.text_type(course.id)
+                    course_string
                 )
                 transaction.rollback()
+                unsuccessful_courses.append(course_string)
+
+            else:
+                COMMAND_LAST_RUN_CACHE.set(course_key)
+                successful_courses.append(course_string)
+
+        return successful_courses, unsuccessful_courses
+
+
+class Command(BaseCommand):
+    """
+    Command to dump modulestore data to neo4j
+
+    Takes the following named arguments:
+      host: the host of the neo4j server
+      port: the port on the server that accepts https requests
+      user: the username for the neo4j user
+      password: the user's password
+
+    Example usage:
+      python manage.py lms dump_to_neo4j --host localhost --port 7473 --user user --password password --settings=aws
+    """
+    def add_arguments(self, parser):
+        parser.add_argument('--host', type=unicode)
+        parser.add_argument('--port', type=int)
+        parser.add_argument('--user', type=unicode)
+        parser.add_argument('--password', type=unicode)
+        parser.add_argument('--courses', type=unicode, nargs='*')
+        parser.add_argument('--override', action='store_true', help='override the cache')
```

Comment:
nit: you should describe this a bit more in the docstring. Or at least reference it in the docstring. +1 for including a help string.